File: lib/features/outbound/presentation/pages/short_term/family_to_family/information/comply_with.dart
// 🐦 Flutter imports:
import 'package:flutter/material.dart';
import 'package:flutter/services.dart';

// 🌎 Project imports:
import 'package:rotary_nl_rye/core/prop.dart';
import 'package:rotary_nl_rye/features/uniform_widgets/back_button.dart';

class ComplyWithPage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        systemOverlayStyle:
            MediaQuery.of(context).platformBrightness == Brightness.light
                ? SystemUiOverlayStyle.dark
                : SystemUiOverlayStyle.light,
        backgroundColor: Colors.transparent,
        elevation: 0.0,
        leading: UniformBackButton(),
        title: Text(
          'Waar moet ik aan voldoen?',
          textScaler: TextScaler.linear(1),
          style: TextStyle(color: Palette.indigo, fontWeight: FontWeight.bold),
        ),
      ),
      body: ListView(
        padding: EdgeInsets.symmetric(horizontal: 20, vertical: 15),
        children: <Widget>[
          Text(
            'Alle jongens en meisjes in de leeftijd van 15 t/m 19* jaar, die open staan voor anderen, van hen willen leren, met hen ervaringen willen uitwisselen, die uit hun eigen vertrouwde omgeving willen stappen en die anderen zonder vooroordelen willen ontmoeten zijn geschikt om aan dit programma deel te nemen.',
            style: TextStyle(fontSize: 15.0),
          ),
          SizedBox(height: 30),
          Center(
            child: Image.asset(
              'assets/image/rotary_blue.png',
              height: 55.0,
            ),
          ),
          SizedBox(height: 15),
          Center(
            child: Text(
              'Update: 8 Sep 2021',
              style: TextStyle(color: Color(0xFF777777)),
            ),
          ),
          SizedBox(height: 60),
        ],
      ),
    );
  }
}

File: rotary-yep-nl/features/student-info/content/outbound/short-term/family-to-family/comply-with.ts
import type { InfoPageContent } from "../../../../types";

export const familyComplyWithContent: InfoPageContent = {
  pageKey: "outbound-family-comply",
  header: {
    icon: "shield-checkmark-outline",
    title: "Waar moet ik aan voldoen?",
    subtitle: "Vereisten en richtlijnen voor Family-to-Family deelnemers",
  },
  sections: [
    {
      id: "eligibility",
      icon: "information-circle-outline",
      title: "Geschiktheid voor deelname",
      blocks: [
        {
          id: "eligibility-card",
          type: "card",
          icon: "people-outline",
          iconColor: "primary",
          accentColor: "primary",
          content:
            "Alle jongens en meisjes in de leeftijd van 15 t/m 19* jaar, die open staan voor anderen, van hen willen leren, met hen ervaringen willen uitwisselen, die uit hun eigen vertrouwde omgeving willen stappen en die anderen zonder vooroordelen willen ontmoeten zijn geschikt om aan dit programma deel te nemen.",
        },
      ],
    },
    {
      id: "age-note",
      icon: "calendar-outline",
      title: "Leeftijdsvereisten",
      blocks: [
        {
          id: "age-note-card",
          type: "card",
          icon: "calendar-outline",
          iconColor: "info",
          accentColor: "info",
          content:
            "*Deelnemers moeten tussen 15 en 19 jaar oud zijn om deel te kunnen nemen aan het Family-to-Family programma.",
        },
      ],
    },
  ],
};
